### PR TITLE
[ko] Localize app name

### PR DESCRIPTION
**Pull request by derornos**
Ref: [Link](https://maruzzing.github.io/study/rnative/React-Native-%EC%95%B1-%EC%9D%B4%EB%A6%84-%EB%8B%A4%EA%B5%AD%EC%96%B4-%EC%84%A4%EC%A0%95%ED%95%98%EA%B8%B0/)

### CODE REVIEW

**Comment by derornos** [comment]
I referenced the documentation in the link above, so it's most likely incorrect code.

**Comment by zzorba** [comment]
I'll give it a shot.

File: android/app/src/main/res/values-ko/strings.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<!-- /values-ko/strings.xml -->
+<resources>
+    <string name="app_name">아컴카드</string>
```
Comment (derornos) [inline android/app/src/main/res/values-ko/strings.xml:3]: It's works.

File: ios/ArkhamCards/ko.lproj/InfoPlist.strings
Comment (derornos) [inline ios/ArkhamCards/ko.lproj/InfoPlist.strings:1]: It doen't works.

Comment (zzorba) [reply ios/ArkhamCards/ko.lproj/InfoPlist.strings:1]: I'm about to go on a vacation, but if you ping me again May 1 I'll figure out how to achieve this.

If you wanted to localize the store copy, email me that and I can add that through the store console as well (no pressure, just mentioning it as an option).

**Pull request merged**